lib/src/main/kotlin/com/swmansion/starknet/data/TypedData.kt

```diff
@@ -11,35 +13,60 @@ import kotlinx.serialization.json.*
  * ```java
  * String typedDataString = """
  * {
- *     "types": {
- *         "StarkNetDomain": [
- *             {"name": "name", "type": "felt"},
- *             {"name": "version", "type": "felt"},
- *             {"name": "chainId", "type": "felt"},
- *         ],
- *         "Person": [
- *             {"name": "name", "type": "felt"},
- *             {"name": "wallet", "type": "felt"},
- *         ],
- *         "Mail": [
- *             {"name": "from", "type": "Person"},
- *             {"name": "to", "type": "Person"},
- *             {"name": "contents", "type": "felt"},
- *         ],
- *     },
- *     "primaryType": "Mail",
- *     "domain": {"name": "StarkNet Mail", "version": "1", "chainId": 1},
- *     "message": {
- *         "from": {
- *             "name": "Cow",
- *             "wallet": "0xCD2a3d9F938E13CD947Ec05AbC7FE734Df8DD826",
- *         },
- *         "to": {
- *             "name": "Bob",
- *             "wallet": "0xbBbBBBBbbBBBbbbBbbBbbbbBBbBbbbbBbBbbBBbB",
- *         },
- *         "contents": "Hello, Bob!",
- *     },
+ *    "types": {
```

Comment:
How was this data obtained? it would be cool to have such info (command) somewhere in here as a comment

Reply:
It was obtained from starknet.js 😄, same as when TypedData was initially added.

Reply:
XD